website/docs/r/datafeed.html.markdown

```diff
@@ -18,6 +18,12 @@ resource "ns1_datasource" "example" {
   sourcetype = "nsone_v1"
 }
 
+resource "ns1_datasource" "example_monitoring" {
+  name       = "example_monitoring"
+  sourcetype = "nsone_monitoring"
+}
+
+
```

Comment:
A single line between resources would match the rest of the examples.

Reply:
fixed